rs/bitcoin/ckbtc/minter/ckbtc_minter.did

```diff
@@ -308,7 +320,7 @@ type MinterInfo = record {
     // This fee is based on the `retrieve_btc_min_amount` setting during canister
     // initialization or upgrades, but may vary according to current network fees.
     retrieve_btc_min_amount : nat64;
-    kyt_fee : nat64;
```

Comment:
@gregorydemay is this a breaking change? Somehow `check_candid_interface_compatibility` passed when I ran `ckbtc_unit_tests` from command line.

Reply:
I tried to run the test locally and it's also successful (I'm sure that the test is executed). This seems weird to me.

Reply:
Locally I can confirm that the following test fails with the expected error: ` Field name mismatch: check_fee and kyt_fee`:
```rust
#[test]
fn check_candid_interface_compatibility() {
    use candid_parser::utils::{service_equal, CandidSource};

    let new_interface = r#"
    type MinterInfo = record {
        check_fee : nat64;
    };

    service : () -> {
       get_minter_info : () -> (MinterInfo) query;
    }
    "#;

    let old_interface = r#"
    type MinterInfo = record {
        kyt_fee : nat64;
    };

    service : () -> {
       get_minter_info : () -> (MinterInfo) query;
    }
    "#;

    service_equal(
        CandidSource::Text(new_interface),
        CandidSource::Text(old_interface),
    )
    .unwrap();
}
```

Reply:
I think I can make sense of it as follows:

1. When run locally `check_candid_interface_compatibility` ensures that the Rust exported Candid interface matches that of the did file. This will not catch breaking changes between different version of the did file.
2. CI will also check that the Candid interface from your branch is compatible with the one in master. This one should fail here, but maybe it didn't because you already had the flag `CI_OVERRIDE_DIDC_CHECK` set?

Reply:
I see, that makes sense. Thanks for digging into it!

To make it backward compatible, I see two choices:
1. keep using `kyt_fee`.
2. introduce `check_fee`, and keep `kyt_fee` but make a comment that it is deprecated.

I kind of prefer 1, because `kyt_fee` is not of an optional type, so we kind of have to keep it forever. WDYT?

Reply:
Yes I would also prefer 1. We could make it less visible in the Rust code by renaming the field as we want and avoid breaking change via `serde_rename`.

Reply:
Thanks for the suggestion! Implemented it as a serde rename.

Reply:
I don't have a strong opinion. Both options are not so great, so I'm okay either way.   :slightly_smiling_face: